datafusion/src/sql/planner.rs

```diff
@@ -2884,11 +2881,9 @@ mod tests {
         let sql = "SELECT order_id, MAX(qty) OVER (ORDER BY order_id), MIN(qty) OVER (ORDER BY order_id DESC) from orders";
         let expected = "\
         Projection: #orders.order_id, #MAX(orders.qty), #MIN(orders.qty)\
-        \n  WindowAggr: windowExpr=[[MAX(#orders.qty)]]\
-        \n    Sort: #orders.order_id ASC NULLS FIRST\
-        \n      WindowAggr: windowExpr=[[MIN(#orders.qty)]]\
-        \n        Sort: #orders.order_id DESC NULLS FIRST\
-        \n          TableScan: orders projection=None";
+        \n  WindowAggr: windowExpr=[[MAX(#orders.qty) ORDER BY [#orders.order_id ASC NULLS FIRST]]]\
```

Comment:
Yeah this is cool to see two different WindowAggr nodes with different ORDER BY clauses. 👍

Reply:
rust uses timsort, and it shall be quick (to reverse an already sorted vec).

ref: https://en.wikipedia.org/wiki/Timsort#Descending_runs